DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: tinygo
+Architecture: all
+Maintainer: @tinygo-org
+Priority: optional
+Version: 0.4.2
```

Comment:
Is this version number adjusted by `fpm` when creating the package?
I think it should be some clearly invalid version (like 0.0.0) which is adjusted to the real version if needed.

Reply:
This would need to be set during the release process.